src/main/scala/scalismo/faces/image/PixelImageNormalization.scala

```diff
@@ -23,18 +23,20 @@ object PixelImageNormalization {
 
   /** Normalizes the image such that its values are on the interval [lower, upper]. */
   def normalizeDoubleImageToRange(image: PixelImage[Double], lower: Double, upper: Double): PixelImage[Double] = {
-    def normalizer(x: Double) = (x - lower) / (upper - lower)
-    if (lower < 0.0 || upper > 1.0)
-      image
-    else
+    require(lower < upper)
```

Comment:
The `require` can be removed as it does not change the correctness. Hence we could invert an image by: `normalizeDoubleImageToRange(image,max,min)`